components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/Oauth2ScopeConstants.java

```diff
@@ -49,6 +49,8 @@
         ERROR_CODE_NOT_AUTHORIZED_ADD_INTERNAL_SCOPE("41008", "User %s is not authorized to add internal scopes"),
         ERROR_CODE_NOT_AUTHORIZED_UPDATE_INTERNAL_SCOPE("41009", "User %s is not authorized to update internal scopes"),
         ERROR_CODE_NOT_AUTHORIZED_DELETE_INTERNAL_SCOPE("41010", "User %s is not authorized to delete internal scopes"),
+        ERROR_CODE_BAD_REQUEST_SCOPE_NAME_NOT_SATIFIED_THE_REGEX("41011", "Scope name is not " +
```

Comment:
If we do not allow changing the regex, it would be user friendly if we say why the scope is invalid rather than saying it should adhere to a specific regex.

Something like "Invalid scope name. A scope name cannot contain special characters x,y or z"

Do we have plans to make this regex configurable?

Reply:
Didn't plan for a config yet.